aries-site/src/pages/templates/code-blocks.mdx

```diff
@@ -0,0 +1,56 @@
+import { Example } from '../../layouts';
+import {
+  CodeBlockExample
+} from '../../examples';
+
+<Example
+  code={[
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/CodeBlocks.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/DarkCodeTheme.js",
+    "https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/code-blocks/LightCodeTheme.js",
+  ]}
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <CodeBlockExample />
+</Example>
+
+
+## Guidance
+
+Code blocks should be created using the [react-syntax-highlighter](https://github.com/react-syntax-highlighter/react-syntax-highlighter) tool.
+
+Prism or Highlighter.js can be used with react-syntax-highlighter with Highlighter.js being the default. Highlighter.js occasionally has trouble highlighting jsx so it is recommended to use Prism instead. 
```

Comment:
```suggestion
Prism or Highlighter.js can be used with react-syntax-highlighter. Highlighter.js is the default, however it can have trouble highlighting JSX, so it is recommended to use Prism instead. 
```